src/openqaoa-core/optimizers/qaoa_optimizer.py

```diff
@@ -0,0 +1,80 @@
+from .training_vqa import (
```

Comment:
why do we call this file `qaoa_optimizer.py` if we are calling the other files `vqa`and the function in this file has nothing specific of qaoa

Reply:
Good point. Maybe the only counter argument here would be that this file requires `QAOAVariationalBaseParams` which is not necessarily generalised for VQAs. However, Ill note that in `training_vqa.py`, `OptimizeVQA` also requires a similar class object for its initialisation input.

I for one, am ok with renaming it. I think `vqa_optimizer` would be appropriate here.

@Q-lds @vishal-ph Quick comments here would help. Thanks.

Reply:
Raised issue in #182 continue discussion there.